src/main/java/com/faforever/client/remote/domain/PartyInfoMessage.java

```diff
@@ -20,11 +20,11 @@ public PartyInfoMessage() {
   public static class PartyMember {
     private Integer player;
     private Boolean ready;
-    private List<Integer> factions;
+    private List<String> factions;
```

Comment:
U know you can do List<Faction> just be parsed correctly. If not you need to make some annotation